src/components/button/button.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Button from './Button';
+
```

Comment:
It is a good practice to wrap tests for a file into 'describe'.
For example:
describe('Button', () => {
  it('should render correctly', () => {
    // your test
  });
});